doc/examples/uuid.rst

```diff
@@ -0,0 +1,472 @@
+.. _handling-uuid-data-example:
+
+Handling UUID Data
+==================
+
+PyMongo ships with built-in support for dealing with UUID types.
+It is trivially simple to store native :class:`uuid.UUID` objects
```

Comment:
I feel like saying "trivially simple" before a wall of text dedicated to the complexity of UUIDs is odd.

Reply:
Changed to 'straightforward'